accounting/src/delta/combine.rs

```diff
@@ -15,30 +15,41 @@
 
 use common::primitives::{signed_amount::SignedAmount, Amount};
 
-use crate::{
-    delta::delta_data_collection::{DataDelta, DeltaMapElement},
-    error::Error,
-};
+use crate::{delta::delta_data_collection::DeltaMapElement, error::Error};
 
 /// Combine data with an element of `DeltaDataCollection`.
 /// An element can be either a Delta or a result of delta undo.
-pub fn combine_data_with_delta<T: Clone>(
+pub fn combine_data_with_delta<T: Clone + PartialEq>(
```

Comment:
Strictly speaking, we only use PartialEq without Eq when it's justified. Otherwise, it's mandatory to use Eq. PartialEq alone can easily become a hidden bug.